style.css

```diff
@@ -395,6 +401,99 @@ textarea:focus {
 
 /* Contact me ends here */
 
+/* Detail Section Starts Here */
+
+.detail-card {
+  display: none;
+}
+
+.detail-card--show {
+  display: flex;
+  position: absolute;
+  z-index: 5;
+  top: 0;
+  left: 0;
+  right: 0;
+}
+
+.detail-card-inner {
+  flex-direction: column;
+  color: #172b4d;
+  background-color: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 1em;
+  justify-content: center;
+  margin: 0.5em 2%;
+  padding: 1em;
+}
```

Comment:
- [ ] Congrats 👏🏽 👏🏽 👏🏽 very good job at the first the modal page looks similar to the Figma design, just a couple of things to correct. When the modal window opens, it should overlap (z-index) the rest of the page and need to be fixed, kindly pay attention to the gray border around the pop-up window and use opacity to have a similar aspect to the Figma design. It is normal that the pop-up window high exceeded the screen size, so we can use `overflow-y: scroll` for a better user experience.